src/index.html

```diff
@@ -2,12 +2,356 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0">
+    <title>Bang & Olufsen</title>
+    <link
+      rel="stylesheet"
+      href="./styles/main.scss">
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Manrope:wght@400;700&display=swap"
+      rel="stylesheet">
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@700&display=swap"
+      rel="stylesheet">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page">
+    <header
+      class="page__header header"
+      id="header">
+      <div class="header__nav ">
+        <img
+          src="images/Logo.svg"
+          alt="Logo"
+          class="header__nav__logo">
+
+        <div class="header__nav__buttons">
```

Comment:
element should belongs only to a block so I could be header__nav-buttons or header__buttons [Typical BEM Mistakes
](https://mate-academy.github.io/fe-program/css/typical-bem-mistakes-en#an-element-of-another-element)
remember to fix it in whole project not only here ;)